### PR TITLE
[3.3.5] [WIP] Addon command machine-readability follow-up

### PR DESCRIPTION
**Pull request by Treeston**
(draft PR)

### CODE REVIEW

**Comment by Shauren** [comment]
https://gist.github.com/Shauren/c696b7383d2a39eca6d6a732babc02b0

Proposed alternative

**Comment by jackpoz** [comment]
which of the 2 versions won ?

**Comment by Shauren** [comment]
We haven't decided - its kind of meaningless without command handlers using it anyway

File: src/server/game/Chat/Chat.h [hunk 1]
```diff
@@ -87,6 +110,40 @@ class TC_GAME_API ChatHandler
...
         }
 
+        template <typename... Args>
+        void AppendMessageData(std::string& str, fmt::internal::NamedArg<char> arg, Args&&... other)
+        {
+            AppendMessageData(str, arg);
+            AppendMessageData(str, std::forward<Args>(other)...); // recurse to append the remaining args
+        }
+        void AppendMessageData(std::string& str, fmt::internal::NamedArg<char> arg);
+        void AppendMessageData(std::string& str) {}
```
Comment (tkrokli) [inline src/server/game/Chat/Chat.h]:
```
In file included from /home/travis/build/TrinityCore/TrinityCore/src/server/game/Battlegrounds/Battleground.cpp:23:
/home/travis/build/TrinityCore/TrinityCore/src/server/game/Chat/Chat.h:120:45:
 fatal error: unused parameter 'str' [-Wunused-parameter]

        void AppendMessageData(std::string& str) {}
                                            ^
```
(followed by 18 similar versions of this error in other files/lines)

**Comment by jackpoz** [comment]
any follow up ?

**Comment by jackpoz** [comment]
Closing this PR as it seems abandoned. Feel free to ask here to reopen it if anyone would like to work on it.

**Pull request closed without merging**